CMakeLists.txt

```diff
@@ -0,0 +1,115 @@
+cmake_minimum_required(VERSION 3.0)
+
+project(KLEE VERSION 1.2.0)
```

Comment:
The minimum CMake versions we need to support is `2.8.12.2` because that's what ships in Ubuntu 14.04LTS.
This means you need to use a different signature for `project()`.

https://cmake.org/cmake/help/v2.8.12/cmake.html#command:project

Something like this

```cmake
project(KLEE C CXX)

################################################################################
# Project version
################################################################################
set(KLEE_VERSION_MAJOR 1)
set(KLEE_VERSION_MINOR 2)
set(KLEE_VERSION_PATCH 0)
set(KLEE_VERSION_TWEAK 0)
set(KLEE_VERSION "${KLEE_VERSION_MAJOR}.${KLEE_VERSION_MINOR}.${KLEE_VERSION_PATCH}.${KLEE_VERSION_TWEAK}")
message(STATUS "KLEE version ${KLEE_VERSION}")
```